molgenis-dataexplorer/src/main/resources/js/dataexplorer-data.js

```diff
@@ -276,14 +276,22 @@
                         createFilter(attr, undefined, undefined, f.id);
                     });
                     if (f.id !== "-" && attr !== undefined) {
-                        info.setTitle(f.id);
-                        info.add('Filter on mutation:', a[0]);
-                        //cache the information
+						if(attr.visible) {
```

Comment:
Does mean that for VCF's the filter no longer works. But looks like this is intended.